ci/integration/logging/run_sg.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/env bash
+# CI script for UBI8 job
+# purpose: spawn sg-core to process messages sent by rsyslog
+
+set -ex
+
+# enable required repo(s)
+cat > /etc/yum.repos.d/fedora-eln.repo <<EOF
+[centos-opstools]
+name=opstools
+baseurl=http://mirror.centos.org/centos/8/opstools/\$basearch/collectd-5/
+gpgcheck=0
+enabled=1
+module_hotfixes=1
+EOF
+
+dnf install -y git golang gcc make qpid-proton-c-devel
+
+export GOBIN=$GOPATH/bin
+export PATH=$PATH:$GOBIN
+
+# install sg-core and start sg-core
+go mod tidy
```

Comment:
Do we need a test at some point that if `go mod tidy` actually results in changes, that we should fail on that?

Reply:
PS not for fixing now, just asking. If you think so, we can just open an issue to add it as an enhancement.

Reply:
Maybe. The reason of tidy is that for some reason without it build script was failing on missing dependencies, because the legacy go path mode was being used and the dependencies were not installed in go path. SO we definitely need some improvements on that side.